### PR TITLE
SSO samples are failing with 6.0 vCenter server. com.vmware.vapi.vcen…

### PR DESCRIPTION
**Pull request by pgbidkar**
SSO samples are failing with 6.0 vCenter server. com.vmware.vapi.vcenter_client.DataCenter module is not present in 6.0. Removed Datacenter listing after connection is successful

Signed-off-by: PavanBidkar <pbidkar@vmware.com>

### CODE REVIEW

**Comment by tianhao64** [comment]
Perhaps we should perform some other API calls instead. 
Maybe create and then delete a tag category?

**Comment by pgbidkar** [comment]
Hi Tianhao,
Yes, I agree. I think tagging is the only this we can verify in 6.0. 
I have modified the samples. Please review.

Added create_category and create_tag methods in both the files. I thought about adding new file in samples.vsphere.common as tagging_helper. Please let me know your thoughts.
Also tried exporting taggingworkflow sample and reuse methods, but methods require changes.

Thanks

**Comment by pgbidkar** [comment]
Tested samples with vSphere 6.0 & 6.5 with Python 2.7.12. Samples are working fine

File: samples/vsphere/sso/embedded_psc_sso_workflow.py [hunk 1]
```diff
@@ -100,14 +105,44 @@ def run(self):
...
+        self.category_svc = Category(stub_config)
+
+        self.category_id = self.create_tag_category('TestTagCat', 'TestTagDesc',
+                                                    CategoryModel.Cardinality.MULTIPLE)
+        assert self.category_id is not None
+        print('Tag category created; Id: {0}'.format(self.category_id))
 
-        datacenter_svc = Datacenter(stub_config)
-        pprint(datacenter_svc.list())
+        self.tag_id = self.create_tag('TestTag', 'TestTagDesc', self.category_id)
```
Comment (tianhao64) [inline samples/vsphere/sso/embedded_psc_sso_workflow.py]: We just need to make a most basic API call to show we login successfully. So I think we can skip the tag creation step and just create a category and then delete it. 

I think it's ok to have the create category method here. No need to create a helper method in common package as we are not using it that often.

Comment (pgbidkar) [inline samples/vsphere/sso/embedded_psc_sso_workflow.py]: Modified sample, to retain only TagCategory creation and deletion. Deleted Tag creation API calls.
Squashed all commits and rebased.
Please review

File: samples/vsphere/sso/embedded_psc_sso_workflow.py [hunk 2]
```diff
@@ -47,6 +46,10 @@ def __init__(self):
         self.session = None
         self.session_id = None
         self.skip_verification = False
+        self.tag_svc = None
```
Comment (tianhao64) [inline samples/vsphere/sso/embedded_psc_sso_workflow.py]: self.tag_svc and tag_id are not needed.

**Comment by pgbidkar** [comment]
Deleted unused variables. Sorry for too many commits.
Please review

**Pull request merged**